scripts/install/SetOfflineTestRunnerConfig.php

```diff
@@ -46,7 +47,12 @@ class SetOfflineTestRunnerConfig extends \common_ext_action_InstallAction
     public function __invoke($params)
     {
         $this->updateTestRunnerConfig();
-        $this->registerOfflineProxy();
+        if (!$this->registerOfflineProxy()) {
+            return new \common_report_Report(
+                \common_report_Report::TYPE_ERROR,
+                "Unable to register the proxy."
+            );
+        }
         $this->disableUnsupportedTestRunnerPlugins();
```

Comment:
If the offline proxy registration fails, we shouldn't disable the unsupported test runner plugins. What do you think?

Reply:
since it returns the report, the disabling of the plugins won't occur